src/models/Profile.ts

```diff
@@ -0,0 +1,5 @@
+export type Reference = {
```

Comment:
Move all profile entities to Profile.ts and use them from here (at the moment this type is not referenced anywhere and overlaps with the one in User.ts)